test_runner/fixtures/utils.py

```diff
@@ -49,7 +49,8 @@ def subprocess_capture(
     echo_stdout=False,
     capture_stdout=False,
     timeout=None,
-    **kwargs: Any,
+    capture_command=True,
```

Comment:
I've added an argument to make this header optional (but enabled by default).
I'm not fully happy with naming, so if you have a better version In mind, please let me know

Reply:
`capture_command` is yeah perhaps a bit too general.

I'll go with the latter (`with_command_header`), can push this.

Reply:
> I'll go with the latter (with_command_header), can push this.

Sure, feel free to take the PR back, thanks!